### PR TITLE
README Update for Submission

### PR DESCRIPTION
**Pull request by superleesa**
- [x] update data directory structure
- [x] add instruction to run the app + train
- [x] (result?)

### CODE REVIEW

**Comment by wiinnie9** [comment]
- [x] UFT and SFT evaluation
- [x] Cite the use of the TOENet - Acknowledgment: This code is based on the [BasicSR] (https://github.com/xinntao/BasicSR) toolbox and [HINet](https://github.com/megvii-model/HINet).
- [x] user guide for training and testing

File: README.md [hunk 1]
```diff
@@ -28,3 +81,44 @@ Data/
...
+2. Navigate to the directory of evaluation using command ```cd evaluation```
+3. Run the command ```python eval.py```.
+4. The output of the average of the metrics would be stored in a yaml file in the save directory.
+
+## Results
+### Metrics Comparison with the Other Models
+![Comparison with the other models](docs/results_1.png)
+
+### Color Histogram Comparison with Other Models
+![Color Histogram](docs/model_comparisons_ver0.2.png)
```
Comment (superleesa) [inline README.md]: i think we can make these collapsable

[make block collapsable](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab)

```suggestion
### Result
<details>
<summary>Metrics Comparison with the Other Models</summary>
![Comparison with the other models](docs/results_1.png)
</details>

<details>
<summary>Color Histogram Comparison with Other Models</summary>
![Color Histogram](docs/model_comparisons_ver0.2.png)
</details>

```

File: README.md [hunk 2]
```diff
@@ -28,3 +81,44 @@ Data/
...
+After the training of the model, we could evaluate the performance of the model using metrics such as PSNR and SSIM.
+1. Open the terminal.
+2. Navigate to the directory of evaluation using command ```cd evaluation```
+3. Run the command ```python eval.py```.
+4. The output of the average of the metrics would be stored in a yaml file in the save directory.
+
+## Results
+### Metrics Comparison with the Other Models
+![Comparison with the other models](docs/results_1.png)
+
```
Comment (superleesa) [inline README.md:72]: can you put comparison of generated images as well pls?

File: image.png
Comment (superleesa) [inline image.png:1]: what's this hahaha

Comment (wiinnie9) [reply image.png:1]: EH SORRY PASTE WRONG HAHAH

**Pull request merged**